go/engine/device_keygen.go

```diff
@@ -168,7 +168,7 @@ func (e *DeviceKeygen) Push(ctx *Context, pargs *DeviceKeygenPushArgs) error {
 
 	ds = e.appendEncKey(ds, ctx, encSigner, eldestKID, pargs.User)
 
-	var pukSigProducer libkb.AggSigProducer = nil
+	var pukSigProducer libkb.AggSigProducer // = nil
```

Comment:
Yeah, though sometimes the lints just change, and it hits the next person who edits the file :p